files/es/web/api/batterymanager/onchargingchange/index.md

```diff
@@ -23,8 +23,8 @@ Donde `battery` es un objeto {{domxref("BatteryManager")}}, y `funcRef` es una f
 ### Contenido HTML
 
 ```html
-<div id="nivel">(nivel de batería desconocido)</div>
-<div id="tiempodeCarga">(tiempo de carga desconocido)</div>
+<div id="level">(nivel de batería desconocido)</div>
+<div id="chargingTime">(tiempo de carga desconocido)</div>
```

Comment:
looks good. Only one question, why is the plain text on parenthesis? It seems unusual to me.

Reply:
I believe this is following the way the sample's done/written in the en-US version. 

The reasoning behind this is *probably (taking a guess here)* that before the `BatteryManager` features were deprecated, the `<div>`s were likely meant to display text dynamically based on the (return) values of those features. Now that `BatteryManager` is essentially unusable, those values are no longer available, so putting the `<div>` text in parentheses kind of "hints" at that — as in the values are now unknown/inaccessible through `BatteryManager` due to deprecation.

Reply:
> looks good. Only one question, why is the plain text on parenthesis? It seems unusual to me.

yes, it is following the sample's written in the en-US, I'll merge it.